Added talks to call for participation (#14)
USRSE/usrse23: pages/participate.md

- **Camera-ready papers / notebooks due**: Monday, June 26, 2023
- **Registration for authors of papers/notebooks**: Monday, June 26, 2023

## Talks

> We invite talks highlighting the above topic areas and the software/software techniques that enabled the research. Accepted talks will be given the opportunity to have a 15 minutes oral presentation during the conference. Talk submissions should be in the form of an abstract, no more than 1 page in length.

Reviewer: ```suggestion
We invite talks highlighting the above topic areas and the software/software techniques that enabled the research.
Accepted talks will be given 15 minutes for an oral presentation during the conference.
Talk submissions should be in the form of an abstract, no more than 1 page in length.
```

Reviewer: I would prefer we do a word limit over a page limit here. We don't define a standard format, font size, etc., so "one page" seems too vague. Can we say 500 words instead? (Or whatever value?)

Author: I do too, but that's what we have said for posters, notebooks, tutorials as well. I'm up for changing it across all categories, not sure though if that's a good idea after we have already published it?

Reviewer: Ehh... We already have it in other places, so okay, let's keep it here, too. But worth noting for next year!

**Timeline**

> - **Submissions open**: Tuesday, February 16, 2023

Reviewer: ```suggestion
- **Submissions open**: Thursday, February 16, 2023
```
